### PR TITLE
DOI-974 registry deployment part and dockerfile changes

### PR DESCRIPTION
**Pull request by KaamilW**

### CODE REVIEW

**Review by ahmedolaibrahim — APPROVED** [review]
Looks good to me

File: devops/helms/iam-org-creator/values.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+iamOrgCreator:
+  config:
+    NESTJS_PORT: 9000
+    NATS_CLIENTS_URL: nats://identityevents-dev-nats.energyweb.org:4222  
+    REQUEST_NEW_ORG_ROLE: mytest.roles.testy.iam.ewc
```
Comment (ahmedolaibrahim) [inline devops/helms/iam-org-creator/values.yaml]: The value of this should be `orgowner.roles.orgcreator.apps.energyweb.iam.ewc`

File: devops/helms/iam-org-creator/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "iam-org-creator.fullname" . }}-configmap
+  labels:
+    {{- include "iam-org-creator.labels" . | nindent 4 }}
+data:
+  NESTJS_PORT: {{ .Values.iamOrgCreator.config.NESTJS_PORT | default "9000" | quote }}
+  NATS_CLIENTS_URL: {{ .Values.iamOrgCreator.config.NATS_CLIENTS_URL | default "nats://identityevents-dev-nats.energyweb.org:4222" | quote }}
+  REQUEST_NEW_ORG_ROLE: {{ .Values.iamOrgCreator.config.REQUEST_NEW_ORG_ROLE | default "mytest.roles.testy.iam.ewc" | quote }}
```
Comment (ahmedolaibrahim) [inline devops/helms/iam-org-creator/templates/configmap.yaml]: the default for this should be`orgowner.roles.orgcreator.apps.energyweb.iam.ewc`.

**Review by ahmedolaibrahim — COMMENTED** [review]
Looks really good @KaamilW. Just some minor details to resolve. Thanks!

**Review by royki — APPROVED** [review]
Seems ok for me.

**Pull request merged**